src/applications/personalization/rated-disabilities/reducers/total-disabilities.js

```diff
@@ -16,16 +16,15 @@ export function totalRating(state = initialState, action) {
         ...state,
         loading: false,
         error: {
-          status: '500',
-          res: 'failed to load', // This changes when the backend gets wired up to the frontend.
+          code: action.response.errors.code,
+          detail: action.response.errors.detail,
```

Comment:
Errors that get returned by an API call come back as an array. These values need to be extracted.
<img width="640" alt="Screen Shot 2019-11-21 at 3 18 04 PM" src="https://user-images.githubusercontent.com/15097156/69385003-d6563380-0c72-11ea-8d2c-20525e4f6567.png">